Readme.md

```diff
@@ -6,7 +6,7 @@
 Use git to clone the repository:
 
 ```bash
-git clone https://github.com/unica-isde/flask-classification-2022-G
```

Comment:
If I accept this change, it will change the readme of my project. I don't think this is a good idea. Could you remove it?